AmplifyPlugins/Predictions/AWSPredictionsPlugin/AWSPredictionsPlugin+Configure.swift

```diff
@@ -19,21 +19,26 @@ extension AWSPredictionsPlugin {
     /// - Throws:
     ///   - PluginError.pluginConfigurationError: If one of the configuration values is invalid or empty
     public func configure(using configuration: Any?) throws {
-
-        guard let jsonValueConfiguration = configuration as? JSONValue else {
+        let predictionsConfiguration: PredictionsPluginConfiguration
+        if configuration is AmplifyOutputsData {
             throw PluginError.pluginConfigurationError(
-                PluginErrorMessage.decodeConfigurationError.errorDescription,
-                PluginErrorMessage.decodeConfigurationError.recoverySuggestion
+                PluginErrorMessage.amplifyOutputsConfigurationNotSupportedError.errorDescription,
+                PluginErrorMessage.amplifyOutputsConfigurationNotSupportedError.recoverySuggestion
+            )
+        } else {
```

Comment:
nit: convert to `else if let` and `else` throws exception

Reply:
Updating here, and all other plugins as well